.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -41,4 +41,5 @@
 - [ ] Tests provided or description of manual testing performed is included in the code or PR.
 - [ ] Release note has been added and follows the [library release note guidelines](https://ddtrace.readthedocs.io/en/stable/contributing.html#Release-Note-Guidelines), or else `changelog/no-changelog` label added.
 - [ ] All relevant GitHub issues are correctly linked.
-- [ ] Backports are identified and tagged with Mergifyio.
+- [ ] Change is easily debuggable and contains logs where appropriate.
```

Comment:
If we can avoid using "easily" to be more precise might be better. Can we hint as to how to make it "easy" to debug?

Also logs->telemetry?